parser.json

```diff
@@ -50,15 +37,15 @@
             },
             {
                 "module": "defusedxml",
-                "input_file": "wheels/shared/defusedxml-0.7.1-py2.py3-none-any.whl"
+                "input_file": "wheels/defusedxml-0.7.1-py2.py3-none-any.whl"
             },
             {
                 "module": "idna",
                 "input_file": "wheels/shared/idna-2.10-py2.py3-none-any.whl"
             },
             {
                 "module": "pdfminer",
-                "input_file": "wheels/py3/pdfminer-20191125-py3-none-any.whl"
+                "input_file": "wheels/pdfminer.six-20211012-py3-none-any.whl"
```

Comment:
Could you run `pre-commit run --all-files` as well?
I wanna make sure if the path here is passing on the script. Somehow, I remember it adds wheels files under `shared`. Thanks!